receiver/prometheusremotewritereceiver/go.mod

```diff
@@ -125,3 +125,9 @@ replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/pdatatest
 replace github.com/open-telemetry/opentelemetry-collector-contrib/pkg/golden => ../../pkg/golden
 
 replace github.com/open-telemetry/opentelemetry-collector-contrib/internal/exp/metrics => ../../internal/exp/metrics
+
+// Temporary solution to unblock the development of target_info. Recently we found a bug in the decompression/compression of snappy requests.
+// The solution is being addressed in https://github.com/open-telemetry/opentelemetry-collector/pull/12825. For this reason we are using the forked version of the collector.
+replace go.opentelemetry.io/collector/config/confighttp => github.com/cardinalhq/opentelemetry-collector/config/confighttp v0.0.0-20250411101659-db48f79d3530
```

Comment:
this cannot be merged as is. The fix must land in core and we can then update to the latest snapshot to adopt it.

Reply:
Make sense. I'm using this approach just to validate that the logic is ok. My idea is to keep it in draft until that PR merged. Then merge this one afterwards

Reply:
Agreed, let's work on the issue upstream before merging this one!